scripts/build-and-optimize.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
```

Comment:
Let's make sure the script bails out on the first error. Please set the flag.

Reply:
I agree. Didn't take into account that it won't bail on errors in between.